addons/mail/static/src/core/web/chatter.js

```diff
@@ -199,13 +202,16 @@ export class Chatter extends Component {
      */
     get toFollowersText() {
         const threadName = this.state.thread.displayName || this.state.thread.name;
+        const followers = this.state.thread.followers.slice(-5).map(({ partner }) =>
+            sprintf('<span title="%(email)s">%(emailWithoutDomain)s</span>', {
+                email: escape(partner.email),
+                emailWithoutDomain: escape(partner.emailWithoutDomain),
+            })
+        );
```

Comment:
- The most important followers to show in the inline listing are the 5 oldest followers, not the 5 most recent.
- When there are more than 5 followers, we should see _"and more"_
- very nit-picking: self follower it can just be _"me"_

Reply:
> When there are more than 5 followers, we should see "and more"

This was not in the spec. Given the space we have, I m not sure we should waste it with this. 
CC @paju-odoo

Reply:
> The most important followers to show in the inline listing are the 5 oldest followers, not the 5 most recent.
I think we should keep the same order as in the followers list, which will change to alphabetical order soon

> When there are more than 5 followers, we should see "and more"
That might be a bit long, we could add "..."? 

> very nit-picking: self follower it can just be "me"
Yes, that is better

Reply:
Done è!